README.md

```diff
@@ -125,6 +125,10 @@ submodule. However, you need to install separately:
     # Format code.
     ./tools/format.py
 
+Before running `./tools/format.py`, make sure you have `yapf` installed for the current Python and `rustfmt` for Rust. They can be installed by:
+    pip install yapf
```

Comment:
You need a new line before here for this to be rendered as a code block.

Reply:
Thanks for the heads up! Added.